imagetest/cmd/manager/main.go

```diff
@@ -36,6 +36,7 @@ var (
 	filter        = flag.String("filter", "", "only run tests matching filter")
 	exclude       = flag.String("exclude", "", "skip tests matching filter")
 	machineType   = flag.String("machine_type", "", "machine type to use for test instances")
+	osUnderTest   = flag.String("os", "linux", "operating system that tests will be run on")
```

Comment:
aside from whether we want to add this flag, i don't think there's any benefit to adding it and not having anything use it. so i don't want to merge this PR until we have some of the implementation added.